std/meta.d

```diff
@@ -955,15 +955,18 @@ unittest
 }
 
 
-// : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : : //
-package:
-
-/*
- * With the builtin alias declaration, you cannot declare
- * aliases of, for example, literal values. You can alias anything
- * including literal values via this template.
+/**
+ * With the builtin alias declaration, you cannot declare aliases of,
```

Comment:
@John-Colvin, we've enforced it in PRs before, but there are still cases of "you" used in Phobos. Many of them read just fine, but I think the issue with personal pronouns in reference documentation is that it generally puts undue focus on the reader, when the object of interest is usually the technical component. In tutorial-style writing, which can of course appear in reference documentation, I think active voice with "I" and "you" is perfectly natural.

Some quick searching around suggests many people find "you" inappropriate in formal writing for being too casual, but I thought this was a little vague and I'm not sure how it applies to Phobos documentation. When it comes to voice, it seems like the active voice is actually encouraged also in technical and scientific writing for being easier to read, but I feel like using the passive voice is justified in situations like this to avoid misplaced focus.

A lot of our reference documentation avoids the issue by using partial or imperative sentences (e.g. "Alias anything, including literal values."), but of course this isn't always an option for in-depth descriptions.

The more I look into it the more unsure I get. We could always judge it on a case by case basis.